modules/mailer/mailer.go

```diff
@@ -32,8 +32,8 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 	msg.SetHeader("To", to...)
 	msg.SetHeader("Subject", subject)
 	msg.SetDateHeader("Date", time.Now())
-	msg.SetBody("text/plain", body)
```

Comment:
We can, but all other instances using the mailer are injecting html templates rendered to string. For instance https://github.com/joshfng/gogs/blob/fix-issue-email-format/modules/mailer/mail.go#L70